subprojects/internal-android-performance-testing/build.gradle.kts

```diff
@@ -10,7 +10,11 @@ repositories {
 dependencies {
     implementation(project(":base-services"))
     implementation(project(":tooling-api"))
-    api("com.android.tools.build:gradle:3.0.0")
+    api("com.android.tools.build:gradle:3.0.0") {
+        // unify the trove4j version we're using
+        exclude(group = libs.trove4j.substringBefore(":"), module = libs.trove4j.substringAfter(":"))
```

Comment:
Please use a metadata rule and not an exclude for this. The dependency of this artifact should be changed to "our" trove4j library. We should then also remove the other trove4j library from the dependency verification file.